src/cli/ops/console_mode.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of clr-boot-manager.
+ *
+ * Copyright © 2016-2018 Intel Corporation
+ *
+ * clr-boot-manager is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1
+ * of the License, or (at your option) any later version.
+ */
+
```

Comment:
TBH the Solus fork needs to add a copyright section to impacted files since The Events, post 2018. Otherwise only Intel is benefiting from protection of their (and my old) copyrights, rather than Solus for current works.